lib/flamegraph.rb

```diff
@@ -47,7 +47,7 @@ def self.generate(filename=nil, opts = {})
 
     embed_resources = (filename && !opts.key?(:embed_resources)) || opts[:embed_resources]
 
-    renderer = Flamegraph::Renderer.new(backtraces)
+    renderer = Flamegraph::Renderer.new(backtraces, opts)
```

Comment:
Passing `opts` here because I added javascript based filters. So that the page can have default filters as it loads up.